### PR TITLE
Carnation Accessibility tasks.

### PR DESCRIPTION
**Pull request by anpolimus**
Fixed issues:

1. Programs and schedules menus should not automatically close.
2. Searchbox: the label "toggle navigation" button isn't really clear what it does. When I press it, a search box and button are opened back at the beginning of the header. Change the aria-label for this button to reflect it's true function such as "toggle search bar" 
3. Search bar should appear after the toggle button at header
4. Header is not the first content on the page
5. Breadcrumb navigation should be right above the main content
6. Code source order not logical for Locations page
7. Code source order not logical for about page
8. Code source order not logical for give page
9. Code source order not logical for accelerator page
10. Code source order not logical for blog page
11. Blog categories such as "community outreach" and "adults" should be after the blog article headlines 
12. On the join page there are three steps listed to select and membership. There should be a non-visual indicator of what step is the current step the user is on. Do this by adding a screen reader only span or by adding an aria-label indicating which step is currently selected
13. When the search box is visible there is then two buttons that close it one by the edit field that reads "toggle search bar" expanded and also the "toggle navigation" button back at the end of the navigation. Should both of those buttons be there?

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
ok to test

File: modules/custom/openy_calc/js/openy_calc_submit.js [hunk 1]
```diff
@@ -12,9 +12,19 @@
...
   };
 
+  /**
+   * This will focus current tab for aria.
+   */
+  Drupal.behaviors.openy_calc_focus = {
+    attach: function (context, settings) {
+      setTimeout(function() {
+        $('a[role="tab"][aria-selected=true]').focus();
+      }, 1000);
```
Comment (podarok) [inline modules/custom/openy_calc/js/openy_calc_submit.js]: why 1000? why not smaller or larger?

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1954/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1954/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1954/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1954/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1954/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1954/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1954/blogWCAG2AAhtmlcs.txt

**Review by podarok — DISMISSED** [review]
Small question before merge

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build1954/

File: modules/custom/openy_calc/js/openy_calc_submit.js (hunk 1 above)
Comment (anpolimus) [reply modules/custom/openy_calc/js/openy_calc_submit.js]: @podarok please review new version

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1958/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1958/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1958/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1958/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1958/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1958/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1958/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build1958/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build1958/

**Review by podarok — APPROVED** [review]

**Pull request merged**